src/plugins/saved_objects_management/public/services/column_service.ts

```diff
@@ -53,5 +53,5 @@ function registerSpacesApiColumns(
   spacesApi: SpacesApi
 ) {
   // Note: this column is hidden for now because no saved objects are shareable. It should be uncommented when at least one saved object type is multi-namespace.
-  // service.setup().register(new ShareToSpaceSavedObjectsManagementColumn(spacesApi.ui));
+  service.setup().register(new ShareToSpaceSavedObjectsManagementColumn(spacesApi.ui));
```

Comment:
@jportner no problem/objection enabling this column?

Reply:
Ah yes, I asked Matt to enable this, thanks for checking!

We should consider changing the spaces list that is rendered in this column in the SOM page to use `behaviorContext="outside-space"`, for consistency with the Data Views Management page. I opened a separate issue for it here: #124155